source/firstprogram-expanding-network.rst

```diff
@@ -181,159 +185,324 @@ With these values defined, ``create24HourSchedules`` can be made. However, when
 
 .. note:: The timings of departure and return are to be made in the 24 hour format.  
 
-The whole main file code is 
+
+Handling Current Locations
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Now, we also need to handle the location of the individual at every step to ensure that the individual is only in contact with the people in the same location. This is done by adding a new ``currentLocation`` parameter in the person class. We modify the constructor to include this parameter.

```

Comment:
Typeset class names with backticks and correct capitalisation: for example, 

> [...] in the ``Person`` class.

Reply:
We haven't used ideas like "constructors" yet, could you say something that would be a little more clear to novice users like:

> In order to implement this, we add a new attribute to the ``Person`` class called ``currentLocation`` which stores the exact current location of the agent. This attribute has to be dynamically updated at every time-step based on the agent's schedule. This is done by adding a new behaviour <add a local link here to the section on agents and behaviours> ``updateCurrentLocation``. This is implemented below.